### PR TITLE
Matin/Chore: Updated email consent text on signup form

### PR DESCRIPTION
**Pull request by matin-binary**
Changes:

-   Updated email consent text on signup form

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [x] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by suthesan-deriv — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]
Welcome back to coding 😄

File: src/components/custom/_signup-new.js [hunk 1]
```diff
@@ -284,9 +291,20 @@ const SignupNew = ({
...
-                {localize(
-                    'By signing up for a Deriv account, you agree to receive occasional updates about our products, services, and events. You can unsubscribe at any time in your account settings.',
-                )}
+                <Localize
+                    translate_text="By pressing “Create demo account”, you confirm that you are 18 or older. You understand that we may use your email address to send you information about Deriv products and services as well as market news. You can always unsubscribe from these emails in your account settings. For more information, please take a look at Deriv’s <0>Security and privacy</0>."
+                    components={[
+                        <StyledLocalizedLink
+                            key={0}
+                            to="/tnc/security-and-privacy.pdf"
+                            size="12px"
```
Comment (sean-binary) [inline src/components/custom/_signup-new.js:300]:
```suggestion
                            size="1.2rem"
```
A small change to support for screens > 1980px for consistency with the `Header` sizes.

Comment (matin-binary) [reply src/components/custom/_signup-new.js]: Sure thing!

**Review by sean-binary — APPROVED** [review]
👍 LGTM

**Pull request merged**